lsp/nls/src/requests/completion.rs

```diff
@@ -265,6 +265,7 @@ fn field_completion<'ast>(
     world: &'ast World,
     path: &[Ident],
 ) -> Vec<CompletionItem<'ast>> {
+    log::info!("field completion for {ast}");
```

Comment:
Are those logs supposed to stay, or were there just a temporary debug thing?

Reply:
No, it's fine, I just wanted to make sure - I know I added a ton of them for debugging and had a hard time hunting them. If they're useful in general it's totally fine keeping them